### PR TITLE
Add Velocity 3.0.0 support

### LINKED ISSUE #1957: "Couldn't pass ProxyInitializeEvent to plan" - Velocity 3.0
Start Velocity 3.0 with Plan

->

```
[00:41:41 ERROR]: Couldn't pass ProxyInitializeEvent to plan
java.lang.NoSuchMethodError: 'void com.velocitypowered.api.command.CommandManager.register(com.velocitypowered.api.command.Command, java.lang.String[])'
        at com.djrapitops.plan.PlanVelocity.registerCommand(PlanVelocity.java:149) ~[?:?]
        at com.djrapitops.plan.PlanVelocity.onEnable(PlanVelocity.java:129) ~[?:?]
        at com.djrapitops.plan.PlanVelocity.onProxyStart(PlanVelocity.java:84) ~[?:?]
        at com.djrapitops.plan.Lmbda$8.execute(Unknown Source) ~[?:?]
        at com.velocitypowered.proxy.event.UntargetedEventHandler$VoidHandler.lambda$buildHandler$0(UntargetedEventHandler.java:47) ~[velocity.jar:3.0.0-SNAPSHOT (git-3b6b73f2-b32)]
        at com.velocitypowered.proxy.event.VelocityEventManager.fire(VelocityEventManager.java:587) ~[velocity.jar:3.0.0-SNAPSHOT (git-3b6b73f2-b32)]
        at com.velocitypowered.proxy.event.VelocityEventManager.lambda$fire$6(VelocityEventManager.java:468) ~[velocity.jar:3.0.0-SNAPSHOT (git-3b6b73f2-b32)]
        at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1130) ~[?:?]
        at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:630) ~[?:?]
        at java.lang.Thread.run(Thread.java:831) [?:?]

```

### Additional information
Plan starts. But i dont know what if any functions are inhibited by this error

### PR DESCRIPTION
**Pull request by Kopo942**
### Your checklist for this pull request
🚨 Please review the [guidelines for contributing](https://github.com/plan-player-analytics/Plan/blob/master/CONTRIBUTING.md#writing-a-good-pull-request) to this repository.

- [x] Make sure your name is added to Contributors file `/Plan/common/src/main/java/com/djrapitops/plan/delivery/rendering/html/Contributors.java`
- [x] If PR:ing locale changes also add a LangCode with your name `/Plan/common/src/main/java/com/djrapitops/plan/settings/locale/LangCode.java`

### Description
- Switched `VelocityCommand` implementation to `SimpleCommand` subinterface
- Removed old & deprecated `text` component handling API in favor of Adventure components

Tested working on Velocity 3.0.0, backwards compatibility tested on Velocity 1.1.8 (console only). Fixes #1957.

### CODE REVIEW

File: Plan/velocity/src/main/java/com/djrapitops/plan/commands/use/VelocityMessageBuilder.java [hunk 1]
```diff
@@ -16,75 +16,76 @@
...
     public MessageBuilder link(String url) {
-        builder = builder.clickEvent(ClickEvent.openUrl(url));
+        builder.clickEvent(ClickEvent.openUrl(url));
         return this;
     }
 
     @Override
     public MessageBuilder command(String command) {
-        builder = builder.clickEvent(ClickEvent.runCommand(command));
+        builder.clickEvent(ClickEvent.runCommand(command));
```
Comment (AuroraLS3) [inline Plan/velocity/src/main/java/com/djrapitops/plan/commands/use/VelocityMessageBuilder.java:57]: Did you test that hover and click events work as intended (For example the confirmation dialog of `/planproxy db remove <uuid>`) - I had an issue at one point where the lack of builder re-assignment caused the last hover and click apply for all text

**Review by AuroraLS3 — APPROVED** [review]
Thanks! Looks good, I added one comment about an unclear change

File: Plan/velocity/src/main/java/com/djrapitops/plan/commands/use/VelocityMessageBuilder.java (hunk 1 above)
Comment (Kopo942) [reply Plan/velocity/src/main/java/com/djrapitops/plan/commands/use/VelocityMessageBuilder.java:57]: Drafted, see Discord

**Comment by Kopo942** [comment]
Events fixed by storing the previous component in a field.

**Review by AuroraLS3 — APPROVED** [review]
Great work!

**Pull request merged**